Formula/dmd.rb

```diff
@@ -98,6 +87,9 @@ def install
     cp_r ["phobos/std", "phobos/etc"], include/"dlang/dmd"
     lib.install Dir["druntime/lib/*", "phobos/**/libphobos2.a"]
 
+    system "which", "dmd"
```

Comment:
Please remove the `which dmd` and `dmd --help` from the `install` method. They're not necessary, and they could cause it to error, depending on how strict your environment is and whether you have a previous `dmd` install. (i.e. the `brew link` step is what populates `/usr/local/bin`, and it takes place after `install()` returns, so if this is your first time installing `dmd`, `which dmd` is expected to fail at this point.)